README.md

```diff
@@ -72,6 +107,8 @@ The markdown file must be written to disk first. It should be possible to use Pa
 
 ## License
 
+GPL v3
```

Comment:
This can be removed since the information is already contained in the license block below.

Reply:
I try to make things super obvious for people so they don't have to read the dense text of the license.